--- configure	2019-12-09 18:58:24.000000000 +0000
+++ configure	2020-05-19 18:20:01.000000000 +0100
@@ -24463,11 +24463,11 @@
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
     { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"libframe\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "libframe") 2>&5
+  ($PKG_CONFIG --exists --print-errors "framel") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_LIBFRAME_CFLAGS=`$PKG_CONFIG --cflags "libframe" 2>/dev/null`
+  pkg_cv_LIBFRAME_CFLAGS=`$PKG_CONFIG --cflags "framel" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -24480,11 +24480,11 @@
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
     { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"libframe\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "libframe") 2>&5
+  ($PKG_CONFIG --exists --print-errors "framel") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_LIBFRAME_LIBS=`$PKG_CONFIG --libs "libframe" 2>/dev/null`
+  pkg_cv_LIBFRAME_LIBS=`$PKG_CONFIG --libs "framel" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -24505,9 +24505,9 @@
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        LIBFRAME_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors --cflags --libs "libframe" 2>&1`
+	        LIBFRAME_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors --cflags --libs "framel" 2>&1`
         else
-	        LIBFRAME_PKG_ERRORS=`$PKG_CONFIG --print-errors --cflags --libs "libframe" 2>&1`
+	        LIBFRAME_PKG_ERRORS=`$PKG_CONFIG --print-errors --cflags --libs "framel" 2>&1`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$LIBFRAME_PKG_ERRORS" >&5
@@ -24659,13 +24659,13 @@

   # end LALSUITE_ADD_FLAGS

-       { $as_echo "$as_me:${as_lineno-$LINENO}: checking for FrLibIni in -lFrame" >&5
-$as_echo_n "checking for FrLibIni in -lFrame... " >&6; }
+       { $as_echo "$as_me:${as_lineno-$LINENO}: checking for FrLibIni in -lframel" >&5
+$as_echo_n "checking for FrLibIni in -lframel... " >&6; }
 if ${ac_cv_lib_Frame_FrLibIni+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lFrame  $LIBS"
+LIBS="-lframel  $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */

@@ -24700,7 +24700,7 @@
 #define HAVE_LIBFRAME 1
 _ACEOF

-  LIBS="-lFrame $LIBS"
+  LIBS="-lframel $LIBS"

 else
   FRAMEL_AVAILABLE="no"
