diff --git a/lalframe/lalframe.pc.in b/lalframe/lalframe.pc.in
index e3f1eb467c..9e2c28d562 100644
--- a/lalframe/lalframe.pc.in
+++ b/lalframe/lalframe.pc.in
@@ -12,6 +12,6 @@ LALFRAME_PYTHON_PATH=@pyexecdir@
 Name: LALFrame
 Description: LAL Frame Library Support
 Version: @VERSION@
-Requires: lal >= @LAL_VERSION@@PC_REQUIRES_FRAMEC@@PC_REQUIRES_FRAMEL@
+Requires.private: lal >= @LAL_VERSION@@PC_REQUIRES_FRAMEC@@PC_REQUIRES_FRAMEL@
 Libs: -L${libdir} -llalframe
 Cflags: -I${includedir}
